### ISSUE TITLE
type object 'Client' has no attribute 'get_pipeline_id'

### ISSUE THREAD

**Issue by deepio-oc** [kubeflow/pipelines#4559]
### What steps did you take:
In our use case, we need to fetch the pipeline based on the pipeline id before we add a version. As per the documentation mentioned here, 
https://kubeflow-pipelines.readthedocs.io/en/latest/source/kfp.client.html?highlight=get_pipeline_id#kfp.Client.get_pipeline_id

I tried to use get_pipeline_id but it fails with the error type object 'Client' has no attribute 'get_pipeline_id'

```
Python 3.8.5 (default, Jul 21 2020, 10:48:26) 
[Clang 11.0.3 (clang-1103.0.32.62)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> import kfp
>>> kfp.Client.get_pipeline_id("test")
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
AttributeError: type object 'Client' has no attribute 'get_pipeline_id'
>>> 
```

I have hit this with KFP 1.0.0 and that time I have workaround it by using a particular commit id for installing. I was hoping this would be included in next version but it is still missing from 1.0.1

### What happened:

KFP version: N/A

KFP SDK version:
```
kfp                      1.0.1
kfp-server-api           1.0.1
```

/kind bug

/area sdk

**Comment by Ark-kun** [comment]
The root cause seems to be https://github.com/kubeflow/pipelines/issues/4409

**Comment by Bobgy** [comment]
That's right, the feature hasn't been released in 1.0.1.
@Ark-kun is there a way we can configure https://kubeflow-pipelines.readthedocs.io/en/latest/source/kfp.client.html?highlight=get_pipeline_id#kfp.Client.get_pipeline_id to show content for KFP 1.0.1?

In the meantime, we should try to release KFP 1.1 ASAP.

**Comment by deepio-oc** [comment]
get_pipeline_id is present in KFP 1.2.0. Closing it.
